spec/Spec/BlsHelpers.cry

```diff
@@ -43,12 +43,19 @@ bls_div x y = toIntegral (div_mod x' y') where
  * NOTE: this Cryptol function requires n >= 2, which is not the same constraint on the Python function,
```

Comment:
todo: fix this doc to reflect your changes

Reply:
Thank you ....

Reply:
Fixed: 1c71b2cdb298ed6f94181fb450ba78de5dc0c0ca